test-configs.yaml

```diff
@@ -1635,16 +1675,56 @@ test_configs:
       - baseline
       - kselftest
 
+  - device_type: imx6qp-sabresd
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx6sll-evk
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx6sx-sdb
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx6ul-14x14-evk
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx6ull-14x14-evk
+    test_plans:
+      - baseline
+      - kselftest
+
   - device_type: imx7d-sdb
     test_plans:
       - baseline
       - kselftest
 
+  - device_type: imx7ulp-evk
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx8mm-ddr4-evk
+    test_plans:
+      - baseline
+      - kselftest
+
   - device_type: imx8mn-ddr4-evk
     test_plans:
       - baseline
       - kselftest
 
+  - device_type: imx8mp-evk
```

Comment:
No baseline results on staging.

Device online in the lab:
https://lavalab.nxp.com/scheduler/device_type/imx8mp-evk

Reply:
Hi gctucker,
I am not sure what is the issue here.  I could not figure out anything looking at our LAVA setup.
For now, I removed the 4 boards that could not be scheduled so that this pull request may be merged.

I need some help on how to debug this.  As the job cannot be seen, I don't know where to debug.  I tried again to run a job (https://lavalab.nxp.com/scheduler/job/35314) for i.mx6sll-evk and it seems to run fine.  
Can you provide few pointers so that I can debug the issue please?

Reply:
Please start a new PR for this device type and we'll discuss it there.  I'm merging this one now as the good devices are ready to be enabled in production today.